src/rebar3_hex_build.erl

```diff
@@ -0,0 +1,232 @@
+-module(rebar3_hex_build). 
```

Comment:
We will turn this into a proper provider in another PR, but felt it was prudent to go ahead make the module.